src/java.base/share/native/libjli/java.c

```diff
@@ -1508,6 +1342,14 @@ ParseArguments(int *pargc, char ***pargv,
             /* java.class.path set on the command line */
             if (JLI_StrCCmp(arg, "-Djava.class.path=") == 0) {
                 _have_classpath = JNI_TRUE;
+            } else if (JLI_StrCmp(arg, "-Djava.awt.headless=true") == 0) {
```

Comment:
Was this processing moved from somewhere else?

Reply:
Hello David, this was in the SelectVersion() method that we removed in this PR https://github.com/openjdk/jdk/pull/20997/files#diff-c3caaf3e347b1a477e2b7278cb6a35da04a597de5632c13351a6e4e5a2924d21L1154

Reply:
Okay I initially assumed I knew what `SelectVersion` did, now it seems it did half a dozen things and we are still doing a large portion of them, but now moved to other locations in the code: splashscreen, headless ...